.github/PULL_REQUEST_TEMPLATE/b9lab_pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+**Instructions: (For B9lab use only!)** Please fill out the following items in brackets `[ ]`, following the [b9lab-processes guideline](/docs/b9lab-processes.md).
+
+Documentation content update for [Module X, Section Y]
+
+This PR [... (elaborate here)]
+
+[(Optional: Further notes for reviewers)]
+
+
+### R1: Internal review (B9lab)
+
+- Technical content review
```

Comment:
Maybe we should change the wording from `Technical content review` to `Technical review`